include/wx/generic/splitter.h

```diff
@@ -378,7 +411,12 @@ class WXDLLIMPEXP_CORE wxSplitterEvent : public wxNotifyEvent
     // data for the different types of event
     union
     {
-        int pos;            // position for SASH_POS_CHANGED event
+        struct
+        {
+            int pos;            // position for SASH_POS_* events
+            int oldSize;        // window size for SASH_POS_UPDATE event
+            int newSize;        // window size for SASH_POS_UPDATE event
+        } sash;
```

Comment:
It's a bit weird to read `sash.oldSize` or `.newSize` above because it looks like it's the size of the sash. Maybe the struct itself should be called `resize` and the fields `sashPos`, `oldSize` and `newSize` respectively?